spec/adapters/alma/bib_spec.rb

```diff
@@ -2,40 +2,73 @@
 require "rails_helper"
 
 RSpec.describe Alma::Bib do
-  let(:bibs) { "991227850000541, 991227840000541, 991227830000541" }
-  let(:one_bib) { "991227850000541" }
-  let(:alma_record) { file_fixture("alma/#{one_bib}.xml").read }
-  let(:alma_marc_record) { MARC::XMLReader.new(StringIO.new(alma_record)).first }
+  let(:bibs) { "991227850000541, 991227840000541, 991227830000541, 99222441306421" }
+  let(:unsuppressed_991227850000541) { "991227850000541" }
```

Comment:
I will update them yes. I had them like that because I used more than one unsuppressed ids but I removed them when I cleaned the spec. I will keep the id on the file name though.